iptables-wrapper-installer.sh

```diff
@@ -61,7 +64,26 @@ else
     altstyle="none"
 fi
 
-if [ "${1:-}" != "--no-sanity-check" ]; then
+no_sanity_check=""
+no_cleanup=""
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+    --no-sanity-check)
+        no_sanity_check=1
+        ;;
+    --no-cleanup)
+        no_cleanup=1
+        ;;
+    *)
+        echo "ERROR: unknown option: $1" 1>&2
+        exit 1
+        ;;
+    esac
+    shift
+done
+
+if [ -z "$no_sanity_check" ]; then
```

Comment:
```suggestion
if [ -z "${no_sanity_check}" ]; then
```

for consistency. (Bash scripts in k8s always do it that way... I think the idea is that you _need_ the braces in some cases (like if there's text that comes directly after the variable expansion with no space between them) and it's better/more consistent to just _always_ use the braces.)

(likewise with no_cleanup below)

Reply:
Done